src/sidebar/services/api.js

```diff
@@ -226,30 +217,70 @@ export class APIService {
         onRequestFinished: store.apiRequestFinished,
       });
 
+    // Define available API calls.
+    //
+    // The type syntax is APICall<Parameters, Body, Result>, where `void` means
+    // no body / empty response.
```

Comment:
I presume this is following a convention of using multi-line comments only for module, method and typedef comments, and single-line for anything else? It looks a little fugly to my eye. Actually, what it makes me wonder is if the APICall defs should be its own module? Is that too much overdesign?

Reply:
> I presume this is following a convention of using multi-line comments only for module, method and typedef comments, and single-line for anything else?

The convention is to use JSDoc (`/** ... */`) to document symbols (methods, classes, functions, variables) and define custom types and `// ...` for everything else.